MetaMask/metamask-extension
# Fix weth->eth direct wrapping allowance issue

Fixes: Issue with trading WETH->ETH throwing an error due to allowance.

Explanation: I left URL logic in this PR to allow for testing but the URL logic does not need to go into production.

Manual testing steps:  
  - Start with a new wallet
  - Trade Eth for Weth
  - Trade back Weth to Eth
  - See that the trade succeeds
